book/website/collaboration/new-community.md

```diff
@@ -0,0 +1,28 @@
+
+The most recent example is the COVID-19 pandemic and related events {cite}`WHO2020Jul` worldwide including the global lockdown that started in February 2020.
+The research communities responded with a sudden surge of hackathons, data modelling projects, task forces, and working groups to find effective and immediate technical solutions to combat this health crisis.
+
+| ![An illustration of 2 people who are working together and discussing a data chart](../figures/new-community.png) |
+|-----------------------------------|
+|An illustration of analytic process. [Royalty free image from Many Pixels](https://www.manypixels.co/gallery/)|
+
+Such action oriented efforts result from a motivation to pull together and combine our scientific knowledge to address unprecedented situations, which is surely an ideal way to move forward.
+Nevertheless, forming effective collaborations is a challenging process, even in the best of times.
+Keeping a team engaged is a continuous activity that requires regular checking in and evaluation.
+Based on each evaluation we can change and improve our workflow and community practices.
+However, when we hastily create a project to tackle an immediate problem at hand, we don’t have the benefit of time to carry out such evaluation.
+Therefore it is even more important to be intentional when creating teams and communities and apply the tried and tested best practices from the beginning.
+
+So, how can we ensure that the teams and research communities we form during the time of urgency are built on the foundation of excellent science and community practices?
+To help researchers address this issue, especially when starting a community or team-oriented projects in a short amount of time (such as a pandemic), we have compiled recommendations drawn from the experiences of community and technical experts.
+In this chapter we discuss the important aspects that helps make a project truly productive, while being successfully community-oriented.
+We also discuss useful ways to tackle technical issue that can arise in a project.
+In the third subchapter, we urge our readers to identify their stakeholders, actively invite them to participate and value the differences that exist in their teams and communities.
+
+We hope that practices recommended in this chapter will give a good starting point for building successful collaboration with your team and community.
+We want to prompt everyone to think critically not just about the technicalities of the work, but also the social environment in which it is being conducted.
+
+*This chapter was originally discussed and written in [an online post](https://www.software.ac.uk/blog/2020-05-26-cw20-speed-blog-bootstrapping-development-team-during-time-crisis) by Malvika Sharan, Yo Yehudi, Colin Sauze, Raniere Silva, Caroline Jay, Robert Haines and Claire Wyatt as a part of the Collaboration Workshop 2020 speed blog series hosted by [Software Sustainability Institute](software.ac.uk/).*
```

Comment:
```suggestion
*This chapter was originally discussed and written in [an online post](https://www.software.ac.uk/blog/2020-05-26-cw20-speed-blog-bootstrapping-development-team-during-time-crisis) by Malvika Sharan, Yo Yehudi, Colin Sauze, Raniere Silva, Caroline Jay, Robert Haines and Claire Wyatt as a part of the [Collaboration Workshop 2020](https://www.software.ac.uk/cw20) speed blog series hosted by [Software Sustainability Institute](https://www.software.ac.uk).*
```

Reply:
The phrasing looks fine with me @malvikasharan, I've just fixed the link and added one to CW20 directly! 💫